shell/bashrc

```diff
@@ -319,7 +324,6 @@ function phpbrew ()
             unset PHPBREW_PATH
             eval `$BIN env`
             __phpbrew_set_path
-            echo "phpbrew is turned off."
```

Comment:
it's a sad truth that we only have an outdated and imcomplete document, the wiki. However, I'll highlight this change in the `CHANGES.md` as well as the release info.